app/views/coronavirus_form/check_answers.html.erb

```diff
@@ -6,10 +6,12 @@
   <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
 <% end %>
 
-<%= render "govuk_publishing_components/components/title", {
-  title: t('check_your_answers.title'),
-  margin_top: 0
-} %>
+<% cache(t('check_your_answers.title')) do %>
```

Comment:
ah ha! [this must be the fragment caching then](https://guides.rubyonrails.org/caching_with_rails.html).

so... how do we invalidate this?
If we bumped the version of govuk_publishing_components would these get auto invalidated?
What if we added css, or a new margin_top attribute?

Reply:
Great question! I didn't consider the publishing components gem being updated, this would've been invalidated when the content changed, but not when the gem changed.

I've made a change to the view fragment cache keys, following what DDH recommends in [this blog post](https://signalvnoise.com/posts/3113-how-key-based-cache-expiration-works), by adding versions in.

Now we can invalidate the cache by changing the cache key version number when we make a change to the view

If the govuk publishing components gem version gets bumped, or the text the component renders changes, this should also invalidate the cache.